  Header header
  string source
  string destination
  sample_package/sample_message data
